src/Chinchilla.Integration/Features/WithApi.cs

```diff
@@ -1,27 +1,27 @@
-using System.Threading;
-using Chinchilla.Api;
-using NUnit.Framework;
+// using System.Threading;
+// using Chinchilla.Api;
+// using NUnit.Framework;
 
-namespace Chinchilla.Integration.Features
-{
-    public class WithApi : WithLogging
-    {
-        public readonly VirtualHost IntegrationVHost = new VirtualHost("integration");
+// namespace Chinchilla.Integration.Features
+// {
+//     public class WithApi : WithLogging
+//     {
+//         public readonly VirtualHost IntegrationVHost = new VirtualHost("integration");
 
-        protected IRabbitAdmin admin;
+//         protected IRabbitAdmin admin;
 
-        [SetUp]
-        public void ResetVirtualHost()
-        {
-            admin = new RabbitAdmin("http://localhost:15672/api");
-            admin.Delete(IntegrationVHost);
-            admin.Create(IntegrationVHost);
-            admin.Create(IntegrationVHost, new User("guest"), Permission.All);
-        }
+//         [SetUp]
+//         public void ResetVirtualHost()
+//         {
+//             admin = new RabbitAdmin("http://localhost:15672/api");
+//             admin.Delete(IntegrationVHost);
+//             admin.Create(IntegrationVHost);
+//             admin.Create(IntegrationVHost, new User("guest"), Permission.All);
+//         }
 
-        protected void WaitForDelivery()
-        {
-            Thread.Sleep(500);
-        }
-    }
-}
\ No newline at end of file
+//         protected void WaitForDelivery()
```

Comment:
@jonrad @robertcoltheart what do you guys think is the best choice here? This is pretty flakey as there are tests that can take longer than 500ms when run on appveyor because appveyor can be slow.